Cargo.toml

```diff
@@ -284,7 +284,7 @@ assert_approx_eq = "1.1.0"
 assert_unordered = "0.1.1"
 async-stream = "0.3"
 async-trait = "0.1.53"
-axum = "0.5.16"
+axum = "0.6.1"
```

Comment:
Any reason we chose `axum` over `hyper`? I see that it's used by the `http_test`, but I'm just wondering why we didn't choose to reuse `hyper` given that all of our other services use it.